src/pass/llvmgen.cpp

```diff
@@ -3,6 +3,7 @@
 #include "llvm/IR/DataLayout.h"
 #include "llvm/IR/Function.h"
 #include "llvm/IR/InstrTypes.h"
+#include "llvm/IR/IntrinsicsNVPTX.h"
```

Comment:
whenever you include CUDA specific headers, make sure to wrap them inside and ifdef checking if ENABLE_CUDA is true